docs/advanced.rst

```diff
@@ -140,6 +140,34 @@ Bootstrap-Flask, simply use the built-in class ``SwitchField()`` instead of
 ``BooleanField()``. See also the example application.
 
 
+.. _inputcustomization:
+
+Form Input Customization
+------------------------
+
+Rendering Label
+~~~~~~~~~~~~~~~
+
+Bootstrap offers control for rendering
```

Comment:
It's better to use the plural form for these paramters and rename `descr_class` to `description_classes`.

Reply:
I could partly improve this to description_class because like label_class and radio_class it is expanded to description_classes in the macro. Otherwise there is a name clash there, so I followed label_class and radio_class which also support multiple classes.